Ratings_MVP/Ratings_MVP/SearachFilmScreen/View/MoviesListViewController.swift

```diff
@@ -128,6 +130,13 @@ final class MoviesListViewController: UIViewController {
         ])
     }
 
+    private func showApiKeyAlert() {
+        showAPIKeyAlert(title: Constants.getAPIKeyTitle, message: Constants.getAPIKeyText) { result in
+            self.presenter?.getKeyChain().saveValueToKeyChain(result, Constants.apiKeyKey)
```

Comment:
```suggestion
          self.presenter?.checkApiKey()
```
внутри сделать проверку, если ключ есть, передать его в нетворк, и вернуться в контроллер, вызвав запрос на получение данных
 если нет, вызвать алерт, вставить туда ключ, сохранить его в нетворк и вызвать запрос, можно посмотреть как у Артема сделано.